File: php/src/Exception/VerificationException.php
<?php

declare(strict_types=1);

namespace Anubis\Exception;

/** A token was not acceptable: bad signature, wrong audience, expired, or a
 * kid this process does not hold. Comes off the offline path, so it carries no
 * request id — nothing was asked of Anubis. */
final class VerificationException extends AnubisException
{
}

File: java/src/main/java/io/github/gsoultan/anubis/VerificationException.java
package io.github.gsoultan.anubis;

/**
 * A token was not acceptable: bad signature, wrong audience, expired, or a kid
 * this process does not hold.
 *
 * <p>Comes off the offline path, so it carries no request id — nothing was
 * asked of Anubis.
 */
public class VerificationException extends AnubisException {
    public VerificationException(String message) {
        super(message);
    }
}
